mirrord/operator/src/crd/session.rs

```diff
@@ -0,0 +1,71 @@
+use chrono::{DateTime, Utc};
+use kube::CustomResource;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(CustomResource, Clone, Debug, Deserialize, Serialize, JsonSchema)]
+#[kube(
+    group = "mirrord.metalbear.co",
+    version = "v1alpha",
+    kind = "MirrordSession"
+)]
+#[serde(rename_all = "camelCase")]
+pub struct MirrordSessionSpec {
```

Comment:
```suggestion
#[derive(CustomResource, Clone, Debug, Deserialize, Serialize, JsonSchema)]
#[kube(
    group = "mirrord.metalbear.co",
    version = "v1alpha",
    kind = "MirrordClusterSession"
)]
#[serde(rename_all = "camelCase")]
pub struct MirrordClusterSessionSpec {
```

Following someone else's request for consistency :sweat_smile: